addons/sale/report/sale_report_templates.xml

```diff
@@ -37,30 +37,30 @@
                 <span t-field="doc.name"/>
             </h2>
 
-            <div class="row col mt32 mb32" id="informations">
-                <div t-if="doc.client_order_ref" class="mr32">
+            <div class="row mt32 mb32" id="informations">
+                <div t-if="doc.client_order_ref" class="mr32 col-auto mt-2 mb-2 compat12mr0">
```

Comment:
`compat12mr0`? Would it not be easier to just force `margin-right: 0` on all `col` of reports as compatibility ?

Reply:
I don't understand why...

Reply:
As discussed, I removed the compat class and the mr32